src/LinAlg/hiopVectorIntRajaCuda.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2017, Lawrence Livermore National Security, LLC.
```

Comment:
I saw you use 2022 in some other files. Do we need to use the new date for all the files?